kalite/control_panel/templates/control_panel/partials/_facility_table.html

```diff
@@ -27,11 +33,12 @@
                     </thead>
                     <tbody>
                         {% for id,facility in facilities.items %}
-                            <tr>
+                            <tr {% if facility.meta_data_in_need %} class="warning" {% endif %}>
                                 <td>
-                                     <a class="facility-name" href="{% url 'facility_management' zone_id=zone_id facility_id=facility.id %}">
-                                        {{ facility.name }}
-                                    </a>
+                                     <a class="facility-name" href="{% url 'facility_management' zone_id=zone_id facility_id=facility.id %}">{{ facility.name }}</a>
+ 					{% if facility.meta_data_in_need %} 
+						<a href="facility/{{ facility.id }}/edit" class="help-tooltip glyphicon glyphicon-pencil notice" data-toggle="tooltip" data-placement="right" title="{% trans "Edit/update information for this facility." %}"></a>
```

Comment:
One last thing! Can use Django URL reverse template tags here:

```
{% url 'facility_form' facility_id=facility.id %}
```
This will then generate the appropriate URL (even if the URL scheme changes in the future).